stdlib/src/collections/list.mojo

```diff
@@ -751,10 +751,11 @@ struct List[T: CollectionElement](CollectionElement, Sized, Boolable):
         return res^
 
     @always_inline
-    fn __getitem__(self, idx: Int) -> T:
+    fn __getitem__(
+        self: Reference[Self, _, _], idx: Int
+    ) -> ref [self.lifetime] T:
         """Gets a copy of the list element at the given index.
```

Comment:
nit.
`Gets a copy of` -> `Gets a reference of`

Reply:
Nice catch!  I went with `Gets the list element at the given index` because that is what the caller actually ends up with after auto-dereferencing.  It could be more detailed but I am not sure how much should get included in the doc string that will likely be seen often and encountered early when someone starts using Mojo.